LayoutTest.xcodeproj/project.pbxproj

```diff
@@ -677,6 +677,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_BITCODE = NO;
```

Comment:
Sounds good. Does this change both LayoutTestBase and LayoutTest to be bitcode disabled?